tests/test_binpacking.py

```diff
@@ -0,0 +1,52 @@
+# test_binpacking.py
```

Comment:
Why are we testing `binpacking`?  This is an external import, not our code.

Reply:
Just some tests to see/show how the library works.

Reply:
While it's interesting, it doesn't really belong in our repo.  If it's really useful, it should go upstream.

Reply:
Pushed another commit to remove that file.